cinema/serializers.py

```diff
@@ -18,7 +17,56 @@ def update(self, instance, validated_data):
             "description", instance.description
         )
         instance.duration = validated_data.get("duration", instance.duration)
+        instance.save()
+        return instance
+
+
+class GenreSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    name = serializers.CharField(max_length=255)
```

Comment:
The `max_length` for the `name` field in the `GenreSerializer` should be 100 to match the `max_length` of 100 in the `Genre` model.